atlp-rwanda/atlp-devpulse-fn
# fixing themes issue

### PR Description

This PR fixes dark mode issues on all the pages both on the applicant side and admin side

### Description of tasks that were expected to be completed

Some pages on both sides of the app couldn't load the dark themes
As an applicant all pages should be able to turn to dark mode as you click the toggle icon
As an admin i should be able to do so too

### How can this been tested?
- Clone repo
- Checkout branch `fix-themes`
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully
- Use apolo server and interact with doc resolvers
